db/migrate/20230224044021_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] According to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), kindly ensure that a `Post` belongs to an `:author` and not a `:user` so you should  refer the user table for the `foreign_key`. i.e.
```ruby
  def change
    add_reference :likes, :author, null: false, foreign_key: `{ to_table: 'users' }`
  end
```

- [ ] After you have implemented all the above changes, kindly ensure that you update your migrations as they will create `author_id` and a corresponding index. You can refer this **[article](https://stackoverflow.com/questions/1992019/how-can-i-rename-a-database-column-in-a-ruby-on-rails-migration)**  to help you how to rename a column